doc/en/api-specific.md

```diff
@@ -1,52 +1,56 @@
-# 方法 / 定制
+# Methods / Customized
 
-better-scroll 还提供了一些定制的方法，专门用来实现某一个 feature 所用。
+better-scroll also supports several customized methods, to implement some specific feature.
 
 ## goToPage(x, y, time, easing)
-   - 参数
-     - {Number} x 横轴的页数
-     - {Number} y 纵轴的页数
-     - {Number} time 动画执行的时间
-     - {Object} easing 缓动函数，一般不建议修改，如果想修改，参考源码中的 ease.js 里的写法
-   - 返回值：无
-   - 作用：当我们做 slide 组件的时候，slide 通常会分成多个页面。调用此方法可以滚动到指定的页面。   
- 
+   - Parameters:
+     - {Number} x, number of horizontal axis page.
+     - {Number} y, number of vertical axis page.
+     - {Number} time, animation duration.
+     - {Object} easing, easing function, generally don't suggest modifying. If you really need to modify, please consult the format of ease.js in source code.
```

Comment:
maybe 'usually' here is more  suitable than 'generally'
usually it's not suggest to be modified.
the same below.